cmd/crio/main.go

```diff
@@ -40,6 +40,11 @@ func validateConfig(config *server.Config) error {
 		return fmt.Errorf("Unrecognized image volume type specified")
 
 	}
+
+	// This needs to match the read buffer size in conmon
+	if config.LogSizeMax >= 0 && config.LogSizeMax < 8192 {
+		return fmt.Errorf("log size max should be negative or >= 8192")
```

Comment:
Should be -1 or >= 8192
Where does this number come from?  Probably should document why such a weird number.

Reply:
There is a comment explaining that it has to >= conmon read buffer.

Reply:
Is there anywhere that we could stick a variable that this code and conmon could both get to?  I think no, but thought I'd ask.